packages/openactive-integration-tests/ARCHITECTURE.md

```diff
@@ -83,5 +84,40 @@ FeatureHelper.describeFeature(module, {
   // The secondary opportunity criteria to use for multiple OrderItem tests
   controlOpportunityCriteria: 'TestOpportunityBookable',
 },
-function (configuration, orderItemCriteria, featureIsImplemented, logger) {
+function (configuration, orderItemCriteriaList, featureIsImplemented, logger) {
 ```
+
+This is what **Feature Helper** helps with:
+
+### Test Matrix
+
+Most tests are run for all Booking Flows in Scope ([Simple Booking Flow](https://openactive.io/open-booking-api/EditorsDraft/#simple-booking-flow) and [Booking Flow with Approval](https://openactive.io/open-booking-api/EditorsDraft/#booking-flow-with-approval)) as well as for all Bookable Opportunity Types in Scope (e.g. ScheduledSession, IndividualFacilityUseSlot, etc). These two dimensions create a matrix of permutations which must be run for each test.
+
+FeatureHelper handles this automatically, running the test for each permutation in the matrix.
+
+If a test does not support an Opportunity Type or Booking Flow, it can be excluded from the matrix via the appropriate `skip-` parameters to `FeatureHelper.describeFeature(..)`.
```

Comment:
Maybe an example here? 
```suggestion
If a test does not support an Opportunity Type or Booking Flow, it can be excluded from the matrix via the appropriate `skip-` parameters to `FeatureHelper.describeFeature(..)`, for example `skipOpportunityTypes: ['IndividualFacilityUseSlot']`.
```